debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/MongoDbReplicaSetAuthTest.java

```diff
@@ -75,6 +76,7 @@ void setupDatabase(MongoDbDeployment mongo, String dbName) {
     }
 
     @Test
+    @Flaky("DBZ-7537")
```

Comment:
I have already opened this https://issues.redhat.com/browse/DBZ-7507 and annotation added with https://github.com/debezium/debezium/pull/5245/commits/7313bfbf2a6545189d6b0213bc68dd15c00ce965. Dunno if it works on the class.

Reply:
thanks for info @mfvitale . Maybe you should push `@Flaky` annotation in a separate PR to allow to merge it soon ASAP and avoid duplicates, then rebase our PRs on top on the main. Will you send a PR for `MongoDbReplicaSetAuthTest.java`?

Reply:
@vjuranek In the agreed document about the `Flaky workflow` is stated that the `flaky` annotation needs to be added in the same PR and with a separate commit. This can lead to this issue. Maybe it better to review the workflow? WDYT @Naros 

In that case, i f you think that your PR it more easy to be merged soon, you can just use  [DBZ-7507](https://issues.redhat.com/browse/DBZ-7507) in you annotation and then I'll adapt to you changes. It is ok for you?

Reply:
@mfvitale well, I noticed that adding `@Flaky` annotation doesn't have any effect right now (until PR #5251 is merged), so it actually doesn't matter when the annotation is added (my intention to add `@Flaky` ASAP was to have CI agreen ASAP)

Reply:
Applied to `main`.